tests/Integration/Bridge/TolerantParser/SourceCodeFilesystem/ScfClassCompletorTest.php

```diff
@@ -134,6 +158,7 @@ public function provideComplete(): Generator
                     'type' => Suggestion::TYPE_CLASS,
                     'name' => 'Clapping',
                     'short_description' => 'Test\Name\Clapping',
+                    'class_import' => 'Test\Name\Clapping',
```

Comment:
This is fine, but don't need to test this on every single test (the array subset check will allow partial data)

Reply:
yep